README.md

```diff
@@ -170,14 +175,105 @@ For admin login, use the username:`admin`. And for the password, run the followi
 kubectl -n devtroncd get secret devtron-secret -o jsonpath='{.data.ACD_PASSWORD}' | base64 -d
 ```
 
-## :muscle: Trusted By
-Devtron has been trusted by the Enterprises and community all across the globe.
+## :memo: Compatibility notes
+
+### Current build: 
+
+- Devtron uses modified version of [argo rollout](https://argoproj.github.io/argo-rollouts/)
+- Application metrics only works for k8s 1.16+
+
+
+## 🦹 Hyperion
+
+<details>
+ <summary> <b> Hyperion is one of devtron's Web-based modules to manage helm apps that can be installed seperately too. It helps you observe, manage and debug the applications deployed through Helm across multiple clusters minimizing Kubernetes Complexities. Please expand this column to find Hyperion's features and to get Started with it:</b></summary>
+<br>
+
+
+## :tada: Features(For Hyperion)
+ 
+<details><summary> <b> Application-level resource grouping for easier Debugging </b></summary>
+<br>
+
+- Hyperion groups your deployed microservices and displays them in a slick UI for easier monitoring or debugging. Access pod logs and resource manifests right from the Hyperion UI and even edit them!
```

Comment:
its not microservices, its deployed kubernetes components

Reply:
Changing it to "Deployed Helm Charts"